### PR TITLE
adding openssl.conf for creating custom csr and cert

### PR DESCRIPTION
**Pull request by lihongan**
@liangxia please help review and merge. Thanks.

### CODE REVIEW

File: routing/openssl.conf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+O = RedHat
+OU = OpenShift QE
+CN = apps
+
+[v3_req]
+basicConstraints = CA:FALSE
+subjectAltName = @alt_names
+
+[alt_names]
+DNS.1 = *.apps.your-cluster.qe.devcluster.openshift.com
```
Comment (liangxia) [inline routing/openssl.conf]: @qinpingli @jhou1 Are we OK to expose the URL ?

Comment (lihongan) [reply routing/openssl.conf]: that's just an example, let me update to `example.com`.

Comment (lihongan) [reply routing/openssl.conf]: Removed this line and just kept `*.example.com`.

**Pull request merged**